src/records/app/bsky/Feed.sol

```diff
@@ -4,56 +4,70 @@ pragma solidity ^0.8.28;
 import "solidity-cbor/tags/ReadCidSha256.sol";
 import "solidity-cbor/ReadCbor.sol";
 import "../../../repo/ReadStrongRef.sol";
+import "../../../repo/ReadReply.sol";
+import "../../../repo/ReadFacet.sol";
+import "../../../repo/ReadEmbed.sol";
 
 using ReadCbor for bytes;
 using ReadCidSha256 for bytes;
 using ReadStrongRef for bytes;
+using ReadReply for bytes;
+using ReadFacet for bytes;
+using ReadEmbed for bytes;
+
+import "hardhat/console.sol";
 
 library AppBsky {
     bytes18 internal constant nsidFeedLike = "app.bsky.feed.like";
     bytes18 internal constant nsidFeedPost = "app.bsky.feed.post";
     bytes20 internal constant nsidFeedRepost = "app.bsky.feed.repost";
 
-    function readFeedLike(bytes memory cborData) internal pure returns (string memory subject) {
+    function readFeedLike(
+        bytes memory cborData
+    ) internal pure returns (string memory subject) {
         (uint32 byteIdx, uint mapLen) = cborData.Map(0);
 
         require(mapLen == 3, "unexpected number of fields");
 
         bytes32 mapKey;
         for (uint8 mapIdx = 0; mapIdx < mapLen; mapIdx++) {
-            (byteIdx, mapKey,) = cborData.String32(byteIdx, 9);
+            (byteIdx, mapKey, ) = cborData.String32(byteIdx, 9);
             if (mapKey == "subject") {
                 (byteIdx, subject) = cborData.readStrongRef(byteIdx);
             } else if (mapKey == "$type") {
                 bytes32 _type;
-                (byteIdx, _type,) = cborData.String32(byteIdx, 18);
+                (byteIdx, _type, ) = cborData.String32(byteIdx, 18);
                 require(_type == nsidFeedLike, "unexpected record type");
             } else if (mapKey == "createdAt") {
                 // createdAt string unused
                 byteIdx = cborData.skipString(byteIdx);
             } else {
                 // TODO: ignore unknown keys?
-                revert("unexpected record key");
+                revert("unexpected record key in feed");
```

Comment:
I really think we can improve that by using custom errors. I will probably get to it!